Experimental/BenchmarkHashes/hashbench.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
Perhaps we want a copyright header here (I note that I didn't put one in the existing `Experimental` test code, but I probably should have (not sure if we can justify adding them as part of this change?  Slightly out of scope but raising a PR just to modify them feels like hassle

Reply:
Done